src/lib/Util.ts

```diff
@@ -11,9 +11,23 @@ export class Util {
     private static dir: string;
 
     public static getObjectsMissingTechId(objectArray: Array<any>): Array<any>{
-        return objectArray.filter(object => (
-            !(object.enxCPQ__TECH_External_Id__c || object.enxB2B__TECH_External_Id__c || object.enxB2B__TECH_External_ID__c)
-        ));
+        return objectArray 
+            ? objectArray.filter(object => (
+                !(object.enxCPQ__TECH_External_Id__c || object.enxB2B__TECH_External_Id__c || object.enxB2B__TECH_External_ID__c)
+            ))
+            : [];
+    }
+
+    public static removeIdFields(objectArray: Array<any>): void{
```

Comment:
Wydaje mi się, że to samo osiągnąłbyś modyfikując funkcję retrieveFieldNames w Describer.ts pisząc przy tym o wiele mniej kodu i nie tworząc dodatkowej funkcji do której się dużo razy odnosisz,

Reply:
Racja :D